### PR TITLE
Feat/add input in swiftui

### PR DESCRIPTION
**Pull request by vromeiro**
## Description

Add input component in SwfitUI
Add hostingViewController in SwiftUI Samples

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Screenshots (if appropriate):
Simulator Screenshot - iPhone 15 Pro Max - 2023-12-26 at 16 10 07

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

**Review by renanmassaroto — APPROVED** [review]

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+        public init(builder: Builder) {
+            self.init()
+            builder(self)
+        }
+
+        // MARK: View SwiftUI
+
+        private var textFieldView: some View {
+            Group {
```
Comment (AcassioM) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift:97]: Vini, aqui, como vc está usando o Group e o .onReceive é igual em ambos vc pode adicionar ele depois do fechamento de chaves de Group.  assim:



`Group {
                if self.parameters.isSecureTextEntry {
                    SecureField(self.parameters.placeholder, text: self.$parameters.text, onCommit: { self.focused = false })
                        .onTapGesture {
                            self.focused = true
                        }
                } else {
                    TextField(self.parameters.placeholder, text: self.$parameters.text, onEditingChanged: { edit in
                        self.focused = edit
                    })
                }
            }
            .onReceive(Just(self.parameters.text), perform: { text in
                let textMask = self.parameters.onMask?(text) ?? text
                if textMask != self.textOld {
                    self.parameters.text = textMask
                    self.textOld = textMask
                    self.parameters.onValueChanged(textMask)
                    self.parameters.errorMessage = ""
                }
            })`

Comment (AcassioM) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift:97]: I rapa... posso ter me enganado, vou testar aqui

Comment (vromeiro) [reply OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift:97]: boa, to testando

File: OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                        .onTapGesture {
+                            self.focused = true
+                        }
+                } else {
+                    TextField(self.parameters.placeholder, text: self.$parameters.text, onEditingChanged: { edit in
+                        self.focused = edit
+                    })
+                }
+            }
+            .onReceive(Just(self.parameters.text), perform: { text in
```
Comment (AcassioM) [inline OceanComponents/Classes/Components SwiftUI/Input/OceanSwiftUI+InputTextField.swift:109]: 💙

**Review by AcassioM — APPROVED** [review]

**Pull request merged**